### ISSUE TITLE
New layout.legend xref and yref features don't default to container but to paper

### ISSUE THREAD

**Issue by GeorgeFlorian** [plotly/plotly.js#6611]
I've seen the PR: https://github.com/plotly/plotly.js/pull/6589 and I got excited for this new feature because I don't need to calculate legend positioning for each view size.

But on this line here: https://github.com/plotly/plotly.js/pull/6589/commits/5c94e46512874ce66ec13c40eddbe889fceadb57#diff-5da3ec3a0416c8572b65cf31fe31dbc248f8791065a6749405076431efea1179R173 it says `dflt: 'container',` with I guess is `default value: "container"` ?

But on this pen: https://codepen.io/hannahker/pen/wvEZZvN , if you comment `"xref": "container",` or `"yref": "container",` or replace the values with `paper` you can see that the default ref is `container`.

Is this intended or not ?

**Comment by archmoj** [comment]
cc: @hannahker

**Comment by hannahker** [comment]
@GeorgeFlorian yes, it is intended for both `xref` and `yref` to default to `paper`. The commit you've linked was updated. you can see the final attribute definition here: https://github.com/plotly/plotly.js/pull/6589/files#diff-5da3ec3a0416c8572b65cf31fe31dbc248f8791065a6749405076431efea1179
